packages/suite/src/actions/suite/suiteActions.ts

```diff
@@ -309,6 +310,10 @@ export const checkFirmwareRevision =
             type: SUITE.DEVICE_FIRMWARE_REVISION_CHECK,
             payload: { isDisabled },
         });
+        dispatch({
+            type: SUITE.DEVICE_FIRMWARE_HASH_CHECK,
+            payload: { isDisabled },
+        });
```

Comment:
this thunk now dispatches two more granular actions.
I find that preferable to having debug-only granular settings + master switch in normal setting, it could lead to  confusion - forgetting to use either of those switches when writing new code.

:thought_balloon: and it doesn't violate with redux philosophy - an action (or thunk) essentially describes an event = "thing that user did", not a "command to change a thing in state". So an action or thunk can do multiple things in state. 
Yeah I just read the tutorial recently :sweat_smile: :muscle: